Content.Client/Fax/AdminUI/AdminFaxWindow.xaml

```diff
@@ -24,6 +24,7 @@
         <Label Text="{Loc admin-fax-stamp-color}" />
         <ColorSelectorSliders Margin="12 0 0 0" Name="StampColorSelector" Color="#BB3232"/>
         <Control MinHeight="10" />
+        <CheckBox Name="LockPageCheckbox" Text="{Loc admin-fax-lock-page}" ToolTip="{Loc admin-fax-lock-page-tooltip}"/>
```

Comment:
Would you mind removing this and replacing it with a margin too while you're here?

Reply:
Are you referring to the control with MinHeight="10"?

Reply:
yes